articles/types-of-styling-in-css/index.md

```diff
@@ -0,0 +1,87 @@
+When it comes to web development for non-techies the first thought that pops into their heads is a deep dive into a dark abyss. Where you magically open your _third eye_ and manage to talk to the computer in a language only the chosen few can understand. However, this is far from reality. Change sides and take it from a techies perspective. The first thoughts that pop up might be "What framework best fits this task?" or "Should I do this from scratch or customize an existing template?". A great number of us look forward to starting a new project. This to us means learning a different concept or re-enforcing an existing one. But what if you decide to build a new webpage from scratch? Do you remember the basics? How are you going to implement your CSS on the webpage? CSS is part of the backbone of any webpage but do you know that it also has its form of styling? Yes we all know about bootstrap and its ease of use but that is a topic for a different day.
+
+### Styling in CSS
+A simple google search will show you that there three different styling methods in CSS each different from the other. Each level has its application and use scenario depending on what gets the job done.
```

Comment:
Very wordy. Should be: "There are 3 different styling methods. Each level of styling is given a different hierarchical priority and is used for different reasons."

Reply:
You should make the distinction between internal and external CSS, seeing as the "internal" is implied for the first two categories but explicit for third.